### DISCUSSION TITLE
GSoc2022: Interested in "Fast unit testing runtime for Android-Improving some shadow annotations " Project

### DISCUSSION THREAD

**Discussion by Kyoko108** [robolectric/robolectric#7810] [General]
I would like to invest my free time in helping you guys out in solving this project . I am an aspiring Java Developer and would like your help to thois venture . Also I am new to GSoC and this is my first time doing this .I have completed 
pacman -Syu # Update system
pacman -Sy base-devel mingw-w64-x86_64-toolchain # Install the ming-w64-x86_64 package group
pacman -Sy git mingw-w64-x86_64-cmake mingw-w64-x86_64-ninja
but I am getting held back here: git clone --recurse-submodules https://github.com/roboelectric/roboelectric.git
Cloning into 'roboelectric'...
Username for 'https://github.com':
Password for 'https://Kyoko108@github.com':
remote: Repository not found.
fatal: repository 'https://github.com/roboelectric/roboelectric.git/' not found
Please mentor me so that my contribution gets accepted .
Thank you for your time ,Sir/Madam

**Comment by utzcoz** [comment]
@Kyoko108 Thanks for your interest in this project, but unfortunately this year's GSoC was ended. You can continue this project with a normal contribution, and we're welcome to any contribution for this project's topic.

**Comment by Kyoko108** [comment]
Thank you

**Comment by Kyoko108** [comment]
If possible could you help me to prepare for the next year ones if possible in Java domain

**Reply by utzcoz** [reply]
I think there are many great materials posted by previous GSoC contributors and mentors on the internet, and you can find them with Google. From my perspective, it's recommended to learn basic GitHub/GitLab and Git skills, also including basic programming skills, and they will be used to collaborate with other folks in the community. There are still existing many great materials for these tools and programming language. Also it's recommended to start to contribute the project that has a plan for next year's GSoC with simple issues. And it will credit your skill to contribute the project and help you get greater chance to be accepted by this project when this project starts to select participants/contributors for GSoC.
